Cython/Utility/MemoryView.pyx

```diff
@@ -679,7 +679,7 @@ cdef memoryview_cwrapper(object o, int flags, bint dtype_is_object, __Pyx_TypeIn
     return result
 
 @cname('__pyx_memoryview_check')
-cdef inline bint memoryview_check(object o):
+cdef inline bint memoryview_check(object o) noexcept:
     return isinstance(o, memoryview)
```

Comment:
I wasn't sure about this and just verified that `isinstance()` really cannot fail for extension types. It does a plain MRO walk, i.e. traverses straight pointers. So this is fine.